fuel/app/classes/controller/test.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+class Controller_Test extends Controller
+{
+  public function action_ci($arg = NULL)
+  {
+    if ($arg !== NULL) {
+      echo "Not NULL!!";
+    } else {
```

Comment:
__[phpmd]__
* The method action_ci uses an else expression. Else is never necessary and you can simplify the code to work without else.